Resources/Prototypes/Recipes/Crafting/Graphs/tiles.yml

```diff
@@ -28,5 +28,37 @@
             # Needs StackType ID
             - material: WoodPlank
               amount: 1
-    - node: woodtile
```

Comment:
You sure bud? Has this removed the woodtile node which is why you got the error.

Reply:
Either I'm totally blind and can't read changelogs of stuff I made or you're right.